pack.go

```diff
@@ -100,12 +100,12 @@ type PackManifestOptions struct {
 // of the manifest to be packed is determined by packManifestVersion
 // (Recommended value: PackManifestVersion1_1_RC4).
 //
-//   - If packManifestVersion is [PackManifestVersion1_1_RC4],
+//   - If packManifestVersion is [PackManifestVersion1_1_RC4]:
 //     artifactType MUST NOT be empty unless opts.ConfigDescriptor is specified.
-//   - If packManifestVersion is [PackManifestVersion1_0],
-//     artifactType will be used as the the config media type unless
-//     opts.ConfigDescriptor is specified. If artifactType is empty,
-//     "application/vnd.unknown.config.v1+json" will be used.
+//   - If packManifestVersion is [PackManifestVersion1_0]:
+//     when opts.ConfigDescriptor is nil, artifactType will be used as the
+//     config media type; when opts.ConfigDescriptor is NOT nil,
+//     artifactType will be ignored.
```

Comment:
The new documentation is not accurate. If `opts.ConfigDescriptor` is `nil` and `artifactType` is empty, "application/vnd.unknown.config.v1+json" is used.

Reply:
I was wondering if it's worth mention the empty artifactType behavior. We don't encourage users to leave `artifactType` empty, right?

Reply:
Yes, we don’t encourage users to have empty `artifactType`. However, we need to be honest to documenting the behavior if the users insist to leave it empty.